src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListCaseHandler.java

```diff
@@ -53,8 +58,19 @@ public ServiceDataResponse<ServiceData> handle(ServiceData serviceData) {
 
         StartEventResponse startEventResponse = coreCaseDataService.startCaseEvent(LIST_CASE, caseId, CASE_TYPE_ASYLUM);
         AsylumCase asylumCase = coreCaseDataService.getCaseFromStartedEvent(startEventResponse);
+        log.info("asylumCase for  Case ID `{}` contains '{}'", caseId, asylumCase.toString());
 
-        updateListCaseHearingDetails(serviceData, asylumCase);
+        boolean isAppealsLocationRefDataEnabled = false;
+        try {
+            isAppealsLocationRefDataEnabled = featureToggler.getValueAsServiceUser(
+                APPEALS_LOCATION_REF_DATA_FEATURE, false);
+        } catch (Exception ex) {
+            log.info("Error occurred during isBailsLocationRefDataEnabled : " + ex);
+        }
```

Comment:
Not sure why you have this try/catch here. Do we expect FeatureToggler::getValueAsServiceUser to throw exception?
In the log message of the catch block, I suppose you mean 'isAppealsLocationRefDataEnabled'?

Reply:
It was used for debug purposes previously, but I have removed it now as there should be minimal risks of it throwing error after the issue has been addressed